src/body.rs

```diff
@@ -49,13 +45,10 @@ impl Body {
 pub fn read_to_string(mut body: Body) -> ::std::io::Result<String> {
     let mut s = String::new();
     match body.reader {
-        Kind::Reader(ref mut reader, _) => {
-            reader.read_to_string(&mut s)
-        }
-        Kind::Bytes(ref mut bytes) => {
-            (&**bytes).read_to_string(&mut s)
+            Kind::Reader(ref mut reader, _) => reader.read_to_string(&mut s),
+            Kind::Bytes(ref mut bytes) => (&**bytes).read_to_string(&mut s),
         }
-    }.map(|_| s)
+        .map(|_| s)
```

Comment:
However, I don't understand the indenting here.

```rust
match foo {
    a => b,
    c => d,
}.map(|_| foo)
```

vs

```rust
match foo {
        a => b,
        c => d,
    }.map(|_| foo)
```

Prefer the original.